docs/IconButton.md

```diff
@@ -0,0 +1,20 @@
+# `<IconButton />`
```

Comment:
Should this move to #121?

Reply:
#121 includes anything related to IconButton in this PR.  If any of that is removed from this PR, aside from documentation, the banner would not function properly.  Do you recommend that any files be removed from this PR?  This documentation is included in #121 already.